common/src/cellular_at_core.c

```diff
@@ -624,9 +639,19 @@ CellularATError_t Cellular_ATHexStrToHex( const char * pString,
 
         for( i = 0; i < strHexLen; i++ )
         {
-            firstNibble = _charToNibble( *p ) << 4;
+            firstNibble = _charToNibble( *p );
```

Comment:
It's bug fix, it will be an error when _charToNibble returning  0xFF to firstNibble, then shit 4 bit to left would be meaningless. because the length of firstNibble  is 8 bit. This change only for the error return from _charToNibble.